data/joeboadiDev.json

```
@@ -0,0 +1,17 @@
+{
+  "name" : "Joe Boadi",
+  "type" : "personal",
+  "bio" : "Ex-allied health, Tech enthusiast, Software Engineer",
+  "links" : [
+      {
+        "name" : "Twitter",
+        "url" : "https://twitter.com/joeboaadiDev_",
```

correct your twitter profile link

```suggestion
        "url" : "https://twitter.com/joeboadiDev_",
```